tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -42,15 +42,21 @@ copp/test_copp.py::TestCOPP::test_trap_config_save_after_reboot:
 #######################################
 decap/test_decap.py::test_decap[ttl=pipe, dscp=pipe]:
   skip:
-    reason: "Not supported on broadcom after 201911 release and mellanox all releases"
+    reason: "Not supported on broadcom after 201911 release, mellanox all releases and cisco-8000 all releases"
     conditions:
-      - "(asic_type in ['broadcom'] and release not in ['201811', '201911']) or (asic_type in ['mellanox'])"
+      - "(asic_type in ['broadcom'] and release not in ['201811', '201911']) or (asic_type in ['mellanox']) or (asic_type in ['cisco-8000'])"
 
 decap/test_decap.py::test_decap[ttl=pipe, dscp=uniform]:
   skip:
-    reason: "Not supported on backend and broadcom before 202012 release"
+    reason: "Not supported on backend, uniform ttl/dscp mode from 201911 release and broadcom before 202012 release"
     conditions:
-      - "(topo_name in ['t1-backend', 't0-backend']) or (asic_type in ['broadcom'] and release in ['201811', '201911'])"
+      - "(topo_name in ['t1-backend', 't0-backend']) or (release in ['201811']) or (asic_type in ['broadcom'] and release in ['201911'])"
+
+decap/test_decap.py::test_decap[ttl=uniform, dscp=pipe]:
+  skip:
+    reason: "Not supported uniform ttl/dscp mode from 201911 release and cisco-8000 all releases"
+    conditions:
+      - "(release in ['201811']) or (asic_type in ['cisco-8000'])"
```

Comment:
I don't understand the reason of this skip.

Reply:
It mix `if "uniform" in supported_ttl_dscp_params.values()` and `if supported_ttl_dscp_params['dscp'] == 'pipe' and duthost.facts['asic_type'] in ['cisco-8000']`